uber/model_checks.py

```diff
@@ -64,8 +64,8 @@ def dealer_address(group):
 def group_paid(group):
     try:
         amount = int(float(group.amount_paid))
-        if amount < 0 or amount > 2000:
-            return 'Amount Paid must be a reasonable number'
+        if amount < 0:
+            return 'Amount Paid must be a number higher than 0'
```

Comment:
Minor quibble: could you change this to say "0 or higher", since "higher than 0" implies that "0" isn't a valid value.